consensus/src/context.rs

```diff
@@ -310,22 +333,49 @@ pub enum BlockChainContextRequest {
 }
 
 pub enum BlockChainContextResponse {
-    /// Blockchain context response.
+    /// A generic Ok response.
+    ///
+    /// Response to:
+    /// - [`BlockChainContextRequest::NewRXVM`]
+    /// - [`BlockChainContextRequest::Update`]
+    /// - [`BlockChainContextRequest::PopBlocks`]
+    /// - [`BlockChainContextRequest::ClearAltCache`]
+    /// - [`BlockChainContextRequest::AddAltChainContextCache`]
+    Ok,
+
+    /// Response to [`BlockChainContextRequest::Context`]
     Context(BlockChainContext),
+
+    /// Response to [`BlockChainContextRequest::CurrentRxVm`]
+    ///
     /// A map of seed height to `RandomX` VMs.
     RxVms(HashMap<usize, Arc<RandomXVm>>),
```

Comment:
### TODO
Why does this return a `HashMap` when the request is `CurrentRxVm`?

I might be misremembering but I think somewhere in RPC we need access to a RandomX VM, would calling `.last()` on this `HashMap` always give us the latest in a `Some`?

Reply:
> Why does this return a HashMap when the request is CurrentRxVm?

It should probably be `CurrentRxVms`, it needs to return more than one so we can compute PoW concurrently.

> I might be misremembering but I think somewhere in RPC we need access to a RandomX VM

You will probably just need to generate a new VM - I don't think you will need the current active one, I may be wrong though I just can't see why you would.